### PR TITLE
Add CoordConv block

### PR DESCRIPTION
**Pull request by blazekadam**
Inspired by https://arxiv.org/pdf/1807.03247.pdf 

@petrbel git gut?

### CODE REVIEW

File: emloop_tensorflow/tests/ops/coord_test.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        coords = get_coord_channels(inputs)
+
+        inputs_shape = (7, 11, 13, 3)
+        coords_np = sess.run(coords, feed_dict={inputs: np.zeros(inputs_shape)})
+        assert coords_np.shape == inputs_shape[:3]+(2,)
+        assert np.all(coords_np[:, :, 0, 0] == -1.)
+        assert np.all(coords_np[:, -1, :, 1] == 1.)
+
+        small_coords_np = sess.run(coords, feed_dict={inputs: np.zeros((1, 3, 3, 3))})
+        small_coords_expected = np.array([[[[-1, -1], [0, -1], [1, -1]],
```
Comment (petrbel) [inline emloop_tensorflow/tests/ops/coord_test.py:25]: nice test

File: emloop_tensorflow/models/conv.py [hunk 2]
```diff
@@ -38,14 +38,14 @@ def compute_pool_amount(encoder_config: Sequence[str]):
...
                 encoder_config: Sequence[str],
-                is_training: Optional[tf.Tensor]=None,
-                activation: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
-                conv_kwargs: Optional[dict]=None,
-                bn_kwargs: Optional[dict]=None,
-                ln_kwargs: Optional[dict]=None,
-                skip_connections: Optional[List[tf.Tensor]]=None,
-                use_bn: bool=False,
-                use_ln: bool=False) -> tf.Tensor:
+                is_training: Optional[tf.Tensor] = None,
```
Comment (petrbel) [inline emloop_tensorflow/models/conv.py:41]: ugly for me... but whatever

**Review by petrbel — APPROVED** [review]
Thanks! From the seminar to implementation just in few hours (including tests)

File: emloop_tensorflow/models/conv.py (hunk 2 above)
Comment (blazekadam) [reply emloop_tensorflow/models/conv.py:41]: Its PEP8 bro..., even pycharm highlights it now

**Pull request merged**